sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventProcessorClientBuilder.java

```diff
@@ -211,6 +212,19 @@ public EventProcessorClientBuilder retry(AmqpRetryOptions retryOptions) {
         return this;
     }
 
+    /**
+     * Sets the client options for the processor client. The application id set on the client options will be used
```

Comment:
I don't see the code change in this PR to use the application id from clientOptions. Is it updated in a different PR?

Reply:
Yeah, this is already done in core amqp.